<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-02-26T08:29:48 IST" name="Suite" finished-at="2024-02-26T08:29:52 IST" duration-ms="3460">
    <groups>
    </groups>
    <test started-at="2024-02-26T08:29:48 IST" name="Login Error Message Test" finished-at="2024-02-26T08:29:52 IST" duration-ms="3460">
      <class name="testObjects.LoginErrorValidationTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:testObjects.LoginErrorValidationTest@503f91c3]" started-at="2024-02-26T08:29:48 IST" name="launchApplication" finished-at="2024-02-26T08:29:51 IST" duration-ms="2434" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="loginErrorValidation()[pri:0, instance:testObjects.LoginErrorValidationTest@503f91c3]" started-at="2024-02-26T08:29:51 IST" name="loginErrorValidation" finished-at="2024-02-26T08:29:51 IST" duration-ms="448" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Login Error Message Validation Failed.  expected [Incorrect email or password.] but found [Login Successfully]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Login Error Message Validation Failed.  expected [Incorrect email or password.] but found [Login Successfully]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at testObjects.LoginErrorValidationTest.loginErrorValidation(LoginErrorValidationTest.java:16)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:481)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1623)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginErrorValidation -->
        <test-method is-config="true" signature="closeApplication()[pri:0, instance:testObjects.LoginErrorValidationTest@503f91c3]" started-at="2024-02-26T08:29:52 IST" name="closeApplication" finished-at="2024-02-26T08:29:52 IST" duration-ms="118" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeApplication -->
      </class> <!-- testObjects.LoginErrorValidationTest -->
    </test> <!-- Login Error Message Test -->
  </suite> <!-- Suite -->
</testng-results>
